src/services/Api.ts

```diff
@@ -59,24 +60,31 @@ const newRequest = <P>(method: HTTP_VERBS, url: string, queryParams: any, data:
     params: queryParams
   });
 
+interface LoginRequest {
+  username: string;
+  password: string;
```

Comment:
Same for `password` as `username`. We should be able to search for `PasswordType` use and not just depend on naming a string consistently.